api/app/signals/apps/api/views/signals/public/signals.py

```diff
@@ -32,12 +30,14 @@ class PublicSignalViewSet(GenericViewSet):
     lookup_field = 'uuid'
     lookup_url_kwarg = 'uuid'
 
-    queryset = Signal.objects.select_related(
-        'category_assignment',
-        'category_assignment__category',
-        'location',
-        'status',
-    ).all()
+    queryset = Signal.objects.all()
+
+    # A queryset based on a materialized view that can quickly provide the information needed for the
+    # geography endpoint
+    geography_queryset = PublicSignalGeographyFeature.objects.exclude(
+        # Only signals that are in an "Open" state
+        Q(state__in=[AFGEHANDELD, AFGEHANDELD_EXTERN, GEANNULEERD, VERZOEK_TOT_HEROPENEN])
```

Comment:
There should be no more Signals in `GESPLITST` is that the case in Amsterdam?

Reply:
I just added it to be sure